code/modules/jobs/job_types/warden.dm

```diff
@@ -10,8 +10,8 @@
 	supervisors = "the head of security"
 	selection_color = "#ffeeee"
 	chat_color = "#EA545E"
-	minimal_player_age = 7
-	exp_requirements = 600
+	minimal_player_age = 14
+	exp_requirements = 900
```

Comment:
warden was fine how it was before. They don't have to do much besides read space law and file crimes. The old requirements were more than enough for that

Reply:
> You should totally give donators a "premium" status with 2x XP boost.
I would be perfectly fine with giving donors free access to all roles, if you want my honest opinion. Players who are directly paying for the server's livelihood are worth believing they aren't here to grief, and probably have a vested interest in not intentionally screwing things up in general.

> how much are we willing to trade the experience of new players for grief protection
I don't see walling off complex jobs as taking way from the new player experience at all - it's a really good thing not to accidentally pick a super-important job when you barely understand basic gameplay. The vast majority of jobs are open after a single day and two hours of play, and only a very small handful of non-command jobs are locked behind the 1 week/ten hours alive lock.

A player who has no idea what choosing "engineer" actually entails probably doesn't appreciate being insulted when they ask "what's the supermatter".